### PR TITLE
[BrokenLinksH2] Fix path in link

### PR DESCRIPTION
**Pull request by DhurataJ**
removed broken link, file not found

### CODE REVIEW

**Comment by DhurataJ** [comment]
@msewaweru - The C+E Skilling team is fixing broken links on docs.microsoft.com. This will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes/removals and does not change other content. This PR is ready for merge. Thank you!

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 1]
```diff
@@ -257,7 +257,7 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 |------------|--------------|
 |Confirm-AzureADDomain|[Confirm-MgDomain](/powershell/module/microsoft.graph.identity.directorymanagement/confirm-mgdomain?view=graph-powershell-1.0&preserve-view=true)|
 |Get-AzureADDomain|[Get-MgDomain](/powershell/module/microsoft.graph.identity.directorymanagement/get-mgdomain?view=graph-powershell-1.0&preserve-view=true)|
-|Get-AzureADDomainNameReference|[Get-MgDomainDomainNameReferenceByRef](/powershell/module/microsoft.graph.identity.directorymanagement/get-mgdomaindomainnamereferencebyref?view=graph-powershell-1.0&preserve-view=true)|
+|Get-AzureADDomainNameReference|Get-MgDomainDomainNameReferenceByRef|
```
Comment (msewaweru) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:260]:
```suggestion
|Get-AzureADDomainNameReference|[Get-MgDomainNameReference](/powershell/module/microsoft.graph.identity.directorymanagement/get-mgdomainnamereference?view=graph-powershell-1.0&preserve-view=true)|
```

File: microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md [hunk 2]
```diff
@@ -471,6 +471,6 @@ The following is the categorization of the Azure AD cmdlets. Select a category t
 |Set-MsolServicePrincipal|[Update-MgServicePrincipal](/powershell/module/microsoft.graph.applications/update-mgserviceprincipal?view=graph-powershell-1.0&preserve-view=true)|
 |Set-MsolUser|[Update-MgUser](/powershell/module/microsoft.graph.users/update-mguser?view=graph-powershell-1.0&preserve-view=true)|
 |Set-MsolUserLicense|[Set-MgUserLicense](/powershell/module/microsoft.graph.users.actions/set-mguserlicense?view=graph-powershell-1.0&preserve-view=true)|
-|Set-MsolUserPassword|[Reset-MgUserAuthenticationMethodPassword](/powershell/module//reset-mguserauthenticationmethodpassword?view=graph-powershell-1.0&preserve-view=true)|
+|Set-MsolUserPassword|[Reset-MgUserAuthenticationMethodPassword](/powershell/module/microsoft.graph.users.actions/reset-mguserauthenticationmethodpassword)|
```
Comment (msewaweru) [inline microsoftgraph/docs-conceptual/azuread-msoline-cmdlet-map.md:474]:
```suggestion
|Set-MsolUserPassword|[Reset-MgUserAuthenticationMethodPassword](/powershell/module/microsoft.graph.users.actions/reset-mguserauthenticationmethodpassword?view=graph-powershell-1.0&preserve-view=true)|
```

**Review by msewaweru — CHANGES_REQUESTED** [review]
Other than the minor edit I missed earlier everything else LTGM. Thanks @DhurataJ. Let me know when ready to merge this.

**Comment by DhurataJ** [comment]
@msewaweru - Feel free to review and merge this PR. Thanks! :-)

**Review by msewaweru — APPROVED** [review]
LGTM! Thanks @DhurataJ.

@timayabi2020, please help me merge this.

**Pull request merged**